pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -187,11 +166,12 @@ var _ = Describe("ProjectService", func() {
 			})
 
 			It("will use a replica number as specified in deploy block", func() {
+				Expect(projectService.Deploy.Replicas).NotTo(BeNil())
```

Comment:
Maybe best to assert that the number of replicas is as expected here?

Reply:
This check is just to ensure that the value set in `BeforeEach` is being picked up, the following assertion verifies the value so I don't think we need to make this more specific IMO, but if you want I can change it